test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1237,6 +1237,113 @@ func Test_RunPodSandbox_MultipleContainersSameVhd_WCOW(t *testing.T) {
 	}
 }
 
+func Test_RunPodSandbox_CoreDump_LCOW(t *testing.T) {
```

Comment:
we should have tests for wcow as well

Reply:
I can't do the same test until https://github.com/microsoft/hcsshim/pull/1087 is in